gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
`expand_paginated_list` should be fine.

The problems with `handled_list` are:
  * it's in the past tense: "handled"
  * it doesn't really say how it's handling the list

changing this to a descriptive present tense verb indicates what's happening when you call it:

```ruby
  @compute_service.handled_list(:servers)
```

vs.

```ruby
  @compute_service.expand_paginated_list(:servers)
```

If `expand` is too weak, you could try `combine` or `collect`.  However, I'd shy away from `collect` since it generally has another meaning in ruby.